src/main/scala/org/apache/spark/ml/linalg/RAPIDSML.scala

```diff
@@ -73,6 +73,28 @@ private[spark] object RAPIDSML extends Serializable {
     jniRAPIDSML.dgemm(transa, transb, m, n, k, alpha, A.values, lda, B.values, ldb, beta, C.values, ldc, deviceID)
   }
 
+  /**
+   *
+   * @param transa int type representing the operation op(A) that is non- or (conj.) transpose.
+   * @param transb int type representing the operation op(B) that is non- or (conj.) transpose.
```

Comment:
This says it's an int type -- what values are expected here?  Curious why this isn't taking CublasOperationT.Value as the parameter type so users know what is accepted and the compiler can help validate.

Reply:
Updated to use CublasOperationT.Value. It's more intuitive.
The original motivation is to avoid passing enum object to JNI directly, and use int as a bridge value. As the GEMM routine need an enum value as well but in native. So I added a [conversion function](https://github.com/NVIDIA/spark-rapids-ml/blob/branch-21.12/native/src/rapidsml_jni.cu#L66-L77 ) to do the job.